erpnext/accounts/report/sales_payment_summary/sales_payment_summary.py

```diff
@@ -118,7 +118,7 @@ def get_mode_of_payment_details(filters):
 	invoice_list_names = ",".join(['"' + invoice['name'] + '"' for invoice in invoice_list])
 	if invoice_list:
 		inv_mop_detail = frappe.db.sql("""select a.owner, a.posting_date,
-			ifnull(b.mode_of_payment, '') as mode_of_payment, sum(b.base_amount) as paid_amount
+			ifnull(b.mode_of_payment, '') as mode_of_payment, sum(b.base_amount - a.base_change_amount) as paid_amount
```

Comment:
As `base_change_amount` is part of parent, if there are multiple mode_of_payments, then it will be deducted multiple times, which is wrong.

Reply:
currently in POS invoices, is it possible to have multiple mode of payments?

Reply:
Yes it is possible to have multiple mode of payments

Reply:
@olamide2, in case of multiple mode of payments for one transaction, we would be assuming that the change amount needs to be adjusted against the cash transaction only or in other words is it fair to assume, that change amount is applicable to Cash Transactions only?

Reply:
Yes you are right  ..change is applicable to cash alone

Reply:
Change has been made to apply change amount adjustment to Cash mode of payment.